### DISCUSSION TITLE
Worker task implementation

### DISCUSSION THREAD

**Discussion by Shivanibhawsar** [Netflix/conductor#2467] [Q&A]
Hi, I am beginner to this. I have my own custom logic to ingest and fetch data from database along with cache. I am implementing Netflix conductor workflow model for this. I want to create worker task to trigger my logic but I am not sure how to integrate worker task with conductor workflow model. How I can integrate my logic with workflow worker task? Could anyone please suggest

**Comment by aravindanr** [comment]
Recommend reading,

1. https://netflix.github.io/conductor/gettingstarted/basicconcepts/
2. https://netflix.github.io/conductor/gettingstarted/client/#worker

**Reply by Shivanibhawsar** [reply]
Hi, I implemented the design based on above mentioned links and it worked well. Now I have created one HTTP Task and added to my workflow. This HTTP task is calling one simple custom REST API. But when I am trying to start my workflow, I am getting below warnings continuously and my Rest API also not triggered. Could you please advise on that?
Below is my HTTP Task - 

 {
      "name": "t1_verify_repo",
      "taskReferenceName": "t1_verify_repo",
      "type": "HttpRequest",
      "http_request": {
        "uri": "http://localhost:8081/v1/add",
        "method": "POST",
        "accept": "/",
        "contentType": "application/json",
        "body": "{\n  \"path\": \"path\",\n  \"contentType\": \"contentType\",\n  \"contentHash\": \"contentHash\",\n  \"modifiedDate\": \"modifiedDate\"\n}",
        "connectionTimeOut": 25,
        "readTimeOut": 25
      }
    }

**Reply by deepakrao666** [reply]
please post ur stack trace here.
